src/ai/backend/agent/kubernetes/kernel.py

```diff
@@ -359,31 +358,29 @@ async def get_repl_out_addr(self) -> str:
         return f"tcp://{self.kernel_host}:{self.repl_out_port}"
 
 
-async def prepare_krunner_env_impl(distro: str, root_path: str) -> Tuple[str, Optional[str]]:
-    if distro.startswith("static-"):
-        distro_name = distro.replace("-", "_")  # pkg/mod name use underscores
-    else:
-        if (m := re.search(r"^([a-z]+)\d+\.\d+$", distro)) is None:
-            raise ValueError('Unrecognized "distro[version]" format string.')
-        distro_name = m.group(1)
+async def prepare_krunner_env_impl(
+    distro: str, entrypoint_name: str, root_path: str
+) -> Tuple[str, Optional[str]]:
     docker = Docker()
     arch = get_arch_name()
     current_version = int(
         Path(
             pkg_resources.resource_filename(
-                f"ai.backend.krunner.{distro_name}", f"./krunner-version.{distro}.txt"
+                f"ai.backend.krunner.{entrypoint_name}", f"./krunner-version.{distro}.txt"
             )
         )
         .read_text()
         .strip()
     )
-    krunner_folder_name = f"backendai-krunner.v{current_version}.{distro}"
+    krunner_folder_name = (  # docker는 {arch}.{distro}임
```

Comment:
If there is no specific reason to do so please write down all the comments in English.

Reply:
Fixed! Sorry about that 😢